ViAn/reportgenerator.cpp

```diff
@@ -32,28 +32,38 @@ void ReportGenerator::create_report() {
         doc->dynamicCall("Add()");
         QAxObject* active_doc = word->querySubObject("ActiveDocument");
```

Comment:
Same here

Reply:
Isn't it only necessary to call delete when using new?

Reply:
Yeah, you're right. My bad